### ISSUE TITLE
datepicker: Cannot select a date using Dragon voice control

### ISSUE THREAD

**Issue by chrispymm** [ministryofjustice/moj-frontend#2218]
A user reported issues navgating the calendar and selecting a date using Dragon.

> I am a fairly new Dragon user and may be doing something 'wrong', but I can't        actually get it to select a date.
  I have tried saying numbers (15, press 15, go to 15, select 15) and also left, right, up, down, next. I tried telling Dragon to press tab, but that just moved the focus to the Select button.
  The only way I can find to select a date using Dragon is to call up the grid view and tell it where to click the mouse, which is a very laborious way of doing things.
  My expectation was that I would say a number (15, 27, whatever) and it would select that date for me.

> Saying 'Monday 16th March' did not work. Saying 'Press right' and 'Press down' is now working, but I expected to be able to just say the number I wanted and would have given up if I was actually trying to use it for something.

Adam Liptrot (accessibility specialist in DWP) suggested the issue may be down to `role=application` on the calendar grid.

This `role` was added in order to fix an issue with NVDA screenreader automatically switching into 'browse' mode on focusing a `button`.

[NVDA issue](https://github.com/ministryofjustice/moj-frontend/issues/1109)

Need to find a solution that resolves both of these issues.  This may involve not using `button` elements for the calendar grid.
